Jenkinsfile.upload_release

```diff
@@ -75,6 +75,10 @@ node ('macos1'){
       }
     }
 
+    stage('Push build tag') {
+      sh ('git push --tags')
```

Comment:
I doubt that we want to push all intermediate tags to repo, we don't really need to have them on GH.

Reply:
Makes sense, I'll push it after the release to Apple & Play store.